src/widget/button.rs

```diff
@@ -213,6 +222,8 @@ impl<'a, S> Button<'a, S> {
 
     builder_methods!{
         pub enabled { enabled = bool }
+        pub hover_color { style.hover_color = Option<Color> }
+        pub clicked_color { style.clicked_color = Option<Color> }
```

Comment:
I don't think the `hover_color` or `clicked_color` methods should take an `Option` argument, as they are already optional to call. Would you mind changing these to take a `Color` directly?